File: backend-integrations-repo/wowsql/connection.mjs
// WowSQL MySQL connection MJS
import mysql from 'mysql2/promise';

export async function getConnection() {
  return await mysql.createConnection({
    host: process.env.WOWSQL_HOST || 'localhost',
    user: process.env.WOWSQL_USER || 'root',
    password: process.env.WOWSQL_PASSWORD || '',
    database: process.env.WOWSQL_DB || 'agent_repo'
  });
}

File: backend-integrations-repo/wowsql/connection.py
# WowSQL/MySQL connection — universal
import os
import mysql.connector

def get_connection():
    return mysql.connector.connect(
        host=os.getenv("WOWSQL_HOST", "localhost"),
        user=os.getenv("WOWSQL_USER", "root"),
        password=os.getenv("WOWSQL_PASSWORD", ""),
        database=os.getenv("WOWSQL_DB", "agent_repo")
    )

# Validated pattern: use registry as source of truth
